### PR TITLE
Added test case where SQL keywords are passed as values

### PR DESCRIPTION
**Pull request by MonilBhavsar**
### Details
Added a couple of tricky test cases which can bypass the query parser 

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/165143

### Tests
All `Write` Tests

_________
**Internal Testing Reminder:** when changing bedrock, please compile auth against your new changes

### CODE REVIEW

File: test/tests/WriteTest.cpp [hunk 1]
```diff
@@ -18,14 +18,15 @@ struct WriteTest : tpunit::TestFixture {
...
+                              TEST(WriteTest::keywordsAsValue),
                               AFTER_CLASS(WriteTest::tearDown)) { }
 
     BedrockTester* tester;
 
     void setup() {
         tester = new BedrockTester({}, {
             "CREATE TABLE foo (bar INTEGER);",
-            "CREATE TABLE stuff (id INTEGER PRIMARY KEY, value INTEGER);",
+            "CREATE TABLE stuff (id INTEGER PRIMARY KEY, value INTEGER, info TEXT);",
```
Comment (flodnv) [inline test/tests/WriteTest.cpp:29]: curious why this change and the one below?

File: test/tests/WriteTest.cpp [hunk 2]
```diff
@@ -176,4 +177,17 @@ struct WriteTest : tpunit::TestFixture {
...
     }
 
+    void keywordsAsValue() {
+        SData query("query: INSERT INTO stuff VALUES ( NULL, 11, 'Please update the test' );");
+        tester->executeWaitVerifyContent(query);
+
+        SData query2("query: INSERT INTO stuff VALUES ( NULL, 12, 'Do not delete the test' );");
+        tester->executeWaitVerifyContent(query2);
+
+        // Monil B - 05/26/21
```
Comment (flodnv) [inline test/tests/WriteTest.cpp:187]: no need for this comment (git blame does this for us 🙃 )

File: test/tests/WriteTest.cpp [hunk 3]
```diff
@@ -176,4 +177,17 @@ struct WriteTest : tpunit::TestFixture {
...
+        // Uncomment this test once https://github.com/Expensify/Expensify/issues/165207 is solved
+        // SData query3("query: UPDATE stuff SET info = 'This is not a where clause';");
+        // tester->executeWaitVerifyContent(query3, "502 Query aborted");
```
Comment (flodnv) [inline test/tests/WriteTest.cpp:190]: Maybe uncomment this test and make it pass with the (unexpected) current response, and leave this commented line explaining why it should be that way instead?

**Review by flodnv — CHANGES_REQUESTED** [review]
Thanks!

File: test/tests/WriteTest.cpp (hunk 1 above)
Comment (MonilBhavsar) [reply test/tests/WriteTest.cpp:29]: We want to include a test where SQL keywords like UPDATE, DELETE are inserted into the table as a values. There was not any column to add TEXT data, so created a column here.
Below in the other query, passed NULL to match the table schema.

File: test/tests/WriteTest.cpp (hunk 2 above)
Comment (MonilBhavsar) [reply test/tests/WriteTest.cpp:187]: Okay removing this.

File: test/tests/WriteTest.cpp (hunk 3 above)
Comment (MonilBhavsar) [reply test/tests/WriteTest.cpp:190]: Yes, can be done. It will also make more sense.

**Comment by MonilBhavsar** [comment]
Updated

**Review by flodnv — APPROVED** [review]
Thanks!!
